src/widgetastic_patternfly5/components/menus/select.py

```diff
@@ -182,3 +183,19 @@ def items(self):
 
 class CheckboxSelect(BaseCheckboxSelect, Dropdown):
     DEFAULT_LOCATOR = './/div[contains(@class, "-c-select")][1]'
+
+
+class BaseTypeaheadSelect(BaseSelect):
+    BUTTON_LOCATOR = (
+        ".//button[(contains(@class, '-c-select__toggle') "
+        "or contains(@class, '-c-menu-toggle')) "
+        "and not(contains(@class, '-c-select__toggle-clear'))]"
```

Comment:
Consider creating a separate locator for the clear button, and a function to click it.